Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -486,6 +486,18 @@ void deinitializeGStreamer()
     teardownVideoEncoderSingleton();
 #endif
 
+    bool isLeaksTracerActive = false;
+    auto activeTracers = gst_tracing_get_active_tracers();
+    while (activeTracers) {
+        auto tracer = adoptGRef(GST_TRACER_CAST(activeTracers->data));
+        if (!isLeaksTracerActive && !g_strcmp0(G_OBJECT_TYPE_NAME(G_OBJECT(tracer.get())), "GstLeaksTracer"))
+            isLeaksTracerActive = true;
+        activeTracers = g_list_delete_link(activeTracers, activeTracers);
+    }
```

Comment:
I would bail out when `isLeaksTracerActive == true` and then do a `g_list_free_full` for the rest of the list to save the creation of the smart ptr, but I guess it is not a big deal as this is done during tear down and maybe tracers are not that many. Your call, no strong opinion.